apps/cardiovascular-patient-app/tsconfig.app.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["src/test.ts", "src/polyfills.ts", "src/**/*.spec.ts"],
+  "include": ["**/*.ts"]
```

Comment:
I don't think we need to repeat the "include" here because it's in ./tsconfig.json

Reply:
Okay, @semicolin I will remove this.